R/select_nse.R

```diff
@@ -47,6 +47,11 @@
     verbose = verbose
   )
 
+  # sanity check, if ".eval_expr" returns a column name, and no index (see #407)
+  if (is.character(selected) && all(selected %in% columns)) {
+    selected <- which(selected %in% columns)
+  }
+
```

Comment:
```r
library(datawizard)
model <- lm(Petal.Length ~ Petal.Width, data=iris)
datawizard::data_select(iris, insight::find_predictors(model, effects = "fixed", flatten = TRUE))
```

@etiennebacher in Dom's example above, `.eval_expr()` now returns a column name, no column index. I added the sanity check here, but maybe it's better placed in `.eval_expr()` - However, I wasn't sure if a character vector can be a valid return value for `.eval_expr()`, so I didn't want to place the code there. Maybe you can check, and probably move it to the best place?